processing/src/main/java/io/druid/segment/DimensionHandlerUtils.java

```diff
@@ -341,4 +341,9 @@ public static Float nullToZero(@Nullable Float number)
   {
     return number == null ? ZERO_FLOAT : number;
   }
+
+  public static Number nullToZero(@Nullable Number number)
+  {
+    return number == null ? ZERO_DOUBLE : number;
```

Comment:
In this proposal https://github.com/druid-io/druid/issues/4349 I saw you planned to add this feature:

> Users will be able to specify default values to replace NULL values at ingestion time.

Do you plan on adding that in a later patch? I'm wondering since this PR assumes zero will always be the default value.

Reply:
yeah, that can be added easily as a separate patch/feature, first aiming at adding proper support for Nulls.